sample.env

```diff
@@ -24,19 +24,15 @@ RSPAMD_PASSWORD=
 MAILSERVER_HOSTNAME=mail
 
 # Mailserver version
-# 1.1-stable : Stable version (v1.1-stable github branch)
-# 1.1-latest : Latest development build (master github branch)
-# Latest development builds have been validated through
-# the CI automation system but they are not meant for
-# deployment in production.
-# 1.1-stable and 1.1-latest are rebuilt weekly on the docker hub but if you
-# have any problem with a new build you can switch to 1.1-stable-fallback
-# The fallback tag is rebuilt monthly.
-MAILSERVER_DOCKER_TAG=1.1-stable
+MAILSERVER_DOCKER_TAG=1.1.1
 
 # Docker volumes parent folder
 VOLUMES_ROOT_PATH=/mnt/docker
 
 # Docker containers restart mode
 # https://docs.docker.com/compose/compose-file/#restart
 RESTART_MODE=unless-stopped
+
+POSTFIXADMIN_DOCKER_TAG=3.2.4
+
+RAINLOOP_DOCKER_TAG=1.14.0
```

Comment:
Move these line up under `MAILSERVER_DOCKER_TAG=1.1.1`. That way all versions are grouped together.

Reply:
Done

Reply:
@SaraSmiseth would you like to merge, or should I?

Reply:
Merged. I'll update the migration wiki page.